src/components/footer/Footer.module.css

```diff
@@ -1,80 +1,80 @@
 :root {
-     --main-color: #062d1d;
-     --secondary-color: #E3AA4D;
-     --support-color: #ffffff;
+  --main-color: #062d1d;
+  --secondary-color: #e3aa4d;
```

Comment:
Its the same color, why did you need to change this?

Reply:
done